Dockerfile

```diff
@@ -141,8 +141,10 @@ ARG ROS_DEPS="apt-transport-https \
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Add carma user
-RUN useradd -m carma && \
+RUN echo "source /home/carma/.base-image/init-env.sh" >> /etc/skel/.bashrc && \
```

Comment:
Just assuming that this `/etc/skel/.bashrc` is used as the template for creating a bashrc when a new user is setup? If that's the case, I can understand the change, but it's also a little bit broken I think? This is a more general solution, but it still has a hard reference into the yet-to-be-created carma user's home directory. That works fine for the carma user, after we've created it, but if we created any other user it'd break on permissions errors trying to access carma's home folder.

Maybe there's benefits to this change aside from allowing us to potentially create additional users down the line?